src/integrations/blocks/abstract-dynamic-block-v3.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Yoast\WP\SEO\Integrations\Blocks;
+
+use Yoast\WP\SEO\Integrations\Integration_Interface;
+
+/**
+ * Dynamic_Block class.
+ */
+abstract class Dynamic_Block_V3 implements Integration_Interface {
+
+	/**
+	 * The name of the block.
+	 *
+	 * @var string
+	 */
+	protected $block_name;
+
+	/**
+	 * The editor script for the block.
+	 *
+	 * @var string
+	 */
+	protected $script;
+
+	/**
+	 * The base path for the block.
+	 *
+	 * @var string
+	 */
+	protected $base_path;
+
+	/**
+	 *  Returns the conditionals based on which this loadable should be active.
+	 *
+	 * @return array<string>
+	 */
+	public static function get_conditionals() {
+		return [];
+	}
+
+	/**
+	 * Initializes the integration.
+	 *
+	 * This is the place to register hooks and filters.
+	 *
+	 * @return void
+	 */
+	public function register_hooks() {
+		\add_action( 'init', [ $this, 'register_block' ], 11 );
```

Comment:
Why is this priority 11? If you know, maybe add a comment about it?

Reply:
it's because at 10 we load the integrations themselves, so we can only hook from within an integration later than that. See also the WordProof integration

Reply:
I added a comment